### PR TITLE
Edit1

### PR DESCRIPTION
**Pull request by Curis-lab**
Hey would you reviews my code

### CODE REVIEW

File: file.tsx [hunk 1]
```diff
@@ -1,4 +1,8 @@
 console.log('hello');
 
 console.log('take some file');
-console.log('take some action');
\ No newline at end of file
+console.log('take some action');
```
Comment (BrowserLoser) [inline file.tsx:4]: may be should remove clg.

File: file.tsx [hunk 2]
```diff
@@ -1,4 +1,8 @@
 console.log('hello');
 
 console.log('take some file');
-console.log('take some action');
```
Comment (BrowserLoser) [inline file.tsx:4]: should add more items.

File: file.tsx [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
\ No newline at end of file
+console.log('take some action');
+
+const file = 'how to learn to code';
+
+const foun = 'take';
```
Comment (BrowserLoser) [inline file.tsx]:
```suggestion
const foun = 'take';
const found = 'take';
```

**Review by BrowserLoser — CHANGES_REQUESTED** [review]
Little bit changes, let it go

File: file.tsx (hunk 2 above)
Comment (Curis-lab) [reply file.tsx:4]: yeap, I will fix it

File: file.tsx (hunk 1 above)
Comment (Curis-lab) [reply file.tsx:4]: random work well

**Review by BrowserLoser — APPROVED** [review]

**Review by Curis-lab — COMMENTED** [review]
finished the remviews

**Pull request merged**